packages/dx-react-grid/docs/reference/local-grouping.md

```diff
@@ -10,7 +10,10 @@ A plugin that performs local grouping and group expanding/collapsing.
 
 ### Properties
 
-none
+Name | Type | Description
```

Comment:
We typically define the default value column in properties even there are no default values at this moment.